Modules/mathmodule.c

```diff
@@ -3306,9 +3306,83 @@ math_comb_impl(PyObject *module, PyObject *n, PyObject *k)
 }
 
 
+/*[clinic input]
+math.as_integer_ratio
+    x: object
+    /
+greatest common divisor of x and y
```

Comment:
This looks like a copy-paste error. Maybe:
```suggestion
Return the rational representation of x as the pair (numerator, denominator).
```

Reply:
Thank you. But this proposition was rejected.